CRITICAL - 2022-02-17 01:53:24 -->  is not a valid rule.
#0 C:\wamp64\www\City_Taxi\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('')
#1 C:\wamp64\www\City_Taxi\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('upass', 'Password', '123456', Array, Array)
#2 C:\wamp64\www\City_Taxi\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(82): CodeIgniter\Controller->validate(Array)
#4 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#5 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#9 {main}
ERROR - 2022-02-17 01:54:13 --> ErrorException: SQLite3::exec(): no such table: user_details in C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 01:54:13 --> SQLite3::exec(): no such table: user_details
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
ERROR - 2022-02-17 01:55:26 --> ErrorException: SQLite3::exec(): no such table: user_details in C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 01:55:26 --> SQLite3::exec(): no such table: user_details
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
ERROR - 2022-02-17 01:55:36 --> ErrorException: SQLite3::exec(): no such table: user_details in C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 01:55:36 --> SQLite3::exec(): no such table: user_details
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
ERROR - 2022-02-17 01:55:45 --> ErrorException: SQLite3::exec(): no such table: user_details in C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 01:55:45 --> SQLite3::exec(): no such table: user_details
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
ERROR - 2022-02-17 01:59:23 --> ErrorException: SQLite3::exec(): no such table: user_details in C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 01:59:23 --> SQLite3::exec(): no such table: user_details
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 02:00:10 --> Class 'CodeIgniter\Database\pdo\Connection' not found
#0 C:\wamp64\www\City_Taxi\system\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('pdo', 'CodeIgniter\\Dat...', Array)
#1 C:\wamp64\www\City_Taxi\system\Database\Config.php(78): CodeIgniter\Database\Database->load(Array, 'default')
#2 C:\wamp64\www\City_Taxi\system\Model.php(97): CodeIgniter\Database\Config::connect('default')
#3 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(89): CodeIgniter\Model->__construct()
#4 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#5 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#9 {main}
CRITICAL - 2022-02-17 02:00:21 --> Class 'CodeIgniter\Database\sqlite\Connection' not found
#0 C:\wamp64\www\City_Taxi\system\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('sqlite', 'CodeIgniter\\Dat...', Array)
#1 C:\wamp64\www\City_Taxi\system\Database\Config.php(78): CodeIgniter\Database\Database->load(Array, 'default')
#2 C:\wamp64\www\City_Taxi\system\Model.php(97): CodeIgniter\Database\Config::connect('default')
#3 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(89): CodeIgniter\Model->__construct()
#4 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#5 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#9 {main}
ERROR - 2022-02-17 02:00:28 --> ErrorException: SQLite3::exec(): no such table: user_details in C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 02:00:28 --> SQLite3::exec(): no such table: user_details
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
ERROR - 2022-02-17 02:02:48 --> Error connecting to the database: SQLite3 error: Unable to open database: unable to open database file
CRITICAL - 2022-02-17 02:02:48 --> Unable to connect to the database.
Main connection [sqlite3]: SQLite3 error: Unable to open database: unable to open database file
#0 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#7 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#8 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#12 {main}
ERROR - 2022-02-17 02:03:12 --> Error connecting to the database: SQLite3 error: Unable to open database: unable to open database file
CRITICAL - 2022-02-17 02:03:12 --> Unable to connect to the database.
Main connection [sqlite3]: SQLite3 error: Unable to open database: unable to open database file
#0 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#7 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#8 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#12 {main}
ERROR - 2022-02-17 02:03:47 --> Error connecting to the database: SQLite3 error: Unable to open database: unable to open database file
CRITICAL - 2022-02-17 02:03:47 --> Unable to connect to the database.
Main connection [sqlite3]: SQLite3 error: Unable to open database: unable to open database file
#0 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#7 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#8 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#12 {main}
ERROR - 2022-02-17 02:05:56 --> Error connecting to the database: SQLite3 error: Unable to open database: unable to open database file
CRITICAL - 2022-02-17 02:05:56 --> Unable to connect to the database.
Main connection [SQLite3]: SQLite3 error: Unable to open database: unable to open database file
#0 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#7 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#8 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#12 {main}
ERROR - 2022-02-17 02:08:59 --> Error connecting to the database: SQLite3 error: Unable to open database: unable to open database file
CRITICAL - 2022-02-17 02:08:59 --> Unable to connect to the database.
Main connection [SQLite3]: SQLite3 error: Unable to open database: unable to open database file
#0 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#7 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#8 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#12 {main}
ERROR - 2022-02-17 02:09:17 --> Error connecting to the database: SQLite3 error: Unable to open database: unable to open database file
CRITICAL - 2022-02-17 02:09:17 --> Unable to connect to the database.
Main connection [SQLite3]: SQLite3 error: Unable to open database: unable to open database file
#0 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#7 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#8 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#12 {main}
ERROR - 2022-02-17 02:09:47 --> ErrorException: SQLite3::exec(): no such table: user_details in C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
CRITICAL - 2022-02-17 02:09:47 --> SQLite3::exec(): no such table: user_details
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\wamp64\\www\\C...', 115, Array)
#1 C:\wamp64\www\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\wamp64\www\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\wamp64\www\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\wamp64\www\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\wamp64\www\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\wamp64\www\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\wamp64\www\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\wamp64\www\City_Taxi\app\Controllers\Users.php(104): CodeIgniter\BaseModel->save(Array)
#10 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\wamp64\www\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\wamp64\www\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\wamp64\www\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\wamp64\\www\\C...')
#15 {main}
